code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -258,7 +258,7 @@
 		return
 	var/list/modules = list()
 	modules.Add(robot_module_types)
-	if((crisis && security_level == SEC_LEVEL_RED) || crisis_override) //Leaving this in until it's balanced appropriately.
+	if(security_level == SEC_LEVEL_RED || SEC_LEVEL_DELTA) //There are no brakes in the combat module train.

```

Comment:
I tried: if (security_level == (SEC_LEVEL_RED || SEC_LEVEL_DELTA)), it did not work. In my testing, the line I am using atm works fine for both red and delta code.